gtk/src/callbacks.h

```diff
@@ -41,7 +41,7 @@
                 g_thread_create((GThreadFunc)(f), (p), TRUE, NULL)
 #endif
 
-void ghb_check_all_depencencies(signal_user_data_t *ud);
+void ghb_check_all_dependencies(signal_user_data_t *ud);
```

Comment:
I'm pretty sure this is an internal function. I'm happy to split code changes from comments. I just need a bit of guidance